src/connectors/configure-related-items/connectConfigureRelatedItems.ts

```diff
@@ -54,7 +54,16 @@ function createOptionalFilter({
   return `${attributeName}:${attributeValue}<score=${attributeScore || 1}>`;
 }
 
-export default (function connectConfigureRelatedItems(renderFn, unmountFn) {
+type ConfigureRelatedItemsConnector = Connector<
+  ConfigureRendererOptions,
+  ConfigureRelatedItemsConnectorParams
+>;
+
+export default (function connectConfigureRelatedItems<
+  TWidgetParams extends {
+    searchParameters: algoliasearchHelper.PlainSearchParameters;
```

Comment:
I'm confused, I think this makes searchParameters a widget param of configure related items, no?

Reply:
@Haroenv I will only find time to work on this pull request next Friday, I will back to you then.

Reply:
I am going to alias to any here. I propose we re-visit this topic once we analyse if searchParameters should be a type of the connector or widget or wtv.